sympy/physics/control/lti.py

```diff
@@ -0,0 +1,181 @@
+            q = self.den * other.den
+            return TransferFunction(p, q, self.var)
+
+        elif isinstance(other, Expr) and other.has(Symbol):
+            return TransferFunction(self.num - self.den*other, self.den, self.var)
+        else:
+            raise ValueError("{} cannot be subtracted from TransferFunction."
+                .format(type(other)))
+
+    def __mul__(self, other):
+        other = _sympify(other)
+        if isinstance(other, (Integer, Float)):
+            if isinstance(other, Float):
+                other = Float(other, 3)
+            return TransferFunction(self.num*other, self.den, self.var)
+
+        elif isinstance(other, TransferFunction):
+            if not self.var == other.var:
+                raise ValueError("Both the Transfer Functions should be anchored "
+                    "with the same variable.")
+            p = self.num * other.num
+            q = self.den * other.den
+            return TransferFunction(p, q, self.var)
+
+        elif isinstance(other, Expr) and other.has(Symbol):
+            # other input is a polynomial.
+            if not other.is_commutative:
+                raise ValueError("Only commutative expressions can be multiplied "
+                    "with a TransferFunction.")
+            return TransferFunction(self.num*other, self.den, self.var)
+        else:
+            raise ValueError("TransferFunction cannot be multiplied with {}."
+                .format(type(other)))
+
+    __rmul__ = __mul__
+
+    def __div__(self, other):
+        other = _sympify(other)
+        if isinstance(other, (Integer, Float)):
+            if other == 0:
+                raise ValueError("TransferFunction cannot be divided by zero.")
+            if isinstance(other, Float):
+                other = Float(other, 3)
+            return TransferFunction(self.num, self.den*other, self.var)
+
+        elif isinstance(other, TransferFunction):
+            if not self.var == other.var:
+                raise ValueError("Both the Transfer Functions should be anchored "
+                    "with the same variable.")
+            p = self.num * other.den
+            q = self.den * other.num
+            return TransferFunction(p, q, self.var)
+
+        elif isinstance(other, Expr) and other.has(Symbol):
+            return TransferFunction(self.num, self.den*other, self.var)
+        else:
+            raise ValueError("TransferFunction cannot be divided by {}.".
+                format(type(other)))
+
+    __truediv__ = __div__
+
+    def __rtruediv__(self, other):
+        return _sympify(other) * self**-1
+
+    __rdiv__ = __rtruediv__
+
+    def __pow__(self, p):
+        p = sympify(p)
+        if not isinstance(p, Integer):
+            raise ValueError("Exponent must be an Integer.")
+        if p == 0:
+            return TransferFunction(1, 1, self.var)
+        elif p > 0:
+            num_, den_ = self.num**p, self.den**p
+        else:
+            p = abs(p)
```

Comment:
Why would you take the absolute value? A power to a negative number should probably raise an error for transfer functions. I'm not sure that is defined.

Reply:
You'll need a test for a negative exponent.

Reply:
I see the tests below. Ignore this comment.

Reply:
> Why would you take the absolute value?

When the exponent `p` is negative, ratio is inverted. I also verified this from python control.